### PR TITLE
release 0.7.0 

### PR DESCRIPTION
**Pull request by AxeldeRomblay**
**0.7.0 (2019-06-26)**

- add support for Mac OS & Windows
- update support for python versions
- improve setup
- add tests
- improve documentation & examples
- minor changes in the package architecture

### CODE REVIEW

**Comment by AxeldeRomblay** [comment]
Before merging, we need to setup the codecov report in the PR (and set a relative threshold under which the check is not successful) ! Also it could be great to check the build of the doc & the upload to PyPI...

File: mlbox/prediction/predictor.py [hunk 1]
```diff
@@ -10,18 +10,20 @@
 import warnings
 import time
 import operator
+import matplotlib
+matplotlib.use('TkAgg')
```
Comment (Henri-Gerard) [inline mlbox/prediction/predictor.py]:
```suggestion
# The following line allow to use matplolib <= 3.0 versions on mac os
matplotlib.use('TkAgg')
```

File: mlbox/preprocessing/drift/rde_cv.py [hunk 2]
```diff
@@ -0,0 +1,345 @@
+# # coding: utf-8
```
Comment (Henri-Gerard) [inline mlbox/preprocessing/drift/rde_cv.py]: This file is completely commented, it should be remove and included in the next version

File: tests/test_mlbox.py [hunk 3]
```diff
@@ -1,28 +1,6 @@
-#!/usr/bin/env python
```
Comment (Henri-Gerard) [inline tests/test_mlbox.py]: Finally, we should remove this test since it is neither a unit test or an integration test

Comment (AxeldeRomblay) [reply tests/test_mlbox.py]: ok

File: mlbox/preprocessing/drift/rde_cv.py (hunk 2 above)
Comment (AxeldeRomblay) [reply mlbox/preprocessing/drift/rde_cv.py]: ok it will be deleted on master but will stay on dev branch...

**Comment by AxeldeRomblay** [comment]
:)

**Pull request merged**